### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by levino** [jsonresume/resume-cli#767]
Hi there. I this project actively maintained? It looks a bit ragged to be honest. I could maybe give it some love, fix the deployment and get the PRs sorted out. Who is in charge here and to they want to continue being maintainer?

**Comment by SethFalco** [comment]
JSON Resume has a few maintainers, but everyone is pretty busy, so JSON Resume doesn't get as much love as it should.

I'm a volunteer maintainer myself. Though, lately, I haven't given time either in favor of other projects like SVGO. I was just planning to spend a few hours each week to JSON Resume to at least review issues and pull requests, so they can be resolved in a timely manner.

[Thomas Davis](https://github.com/thomasdavis) is the lead maintainer, so if you'd like to hit someone up, they're your go to. Meanwhile, if you do any PRs, you're welcome to mention me, and I'd be happy to review it.

**Comment by levino** [comment]
Thank you for your feed back. Instead of opening any PRs, I would rather want to help merging / closing the ones that are open today. For example what about https://github.com/jsonresume/resume-cli/pull/751 ? Should be a low hanging fruit to approve an merge. If these PRs are annoying, how about disabling depandabot?

**Comment by SethFalco** [comment]
Open issues and pull requests are what I had in mind when I said I'd like to give JSON Resume some time. (Including the dependencies!)

I'd already started reviewing some chores yesterday, but I won't open a PR until this weekend. I'm doing it locally since upgrading the direct dependencies will inherently resolve most of the other ones, so it's a bit cleaner imo. Then I can proceed with doing the ones that impact the lock file only after.

> Instead of opening any PRs

I said that since it can be nice to have visibility of how you'd contribute before inviting you to be a collaborator. I was clarifying that as you're interested, I'd be happy to work with you so other maintainers who do have permission to invite members might be more inclined based on your contributions.

PS: I do have write-access to the repos, but not admin access on the org. I wouldn't be able to invite you myself.

**Comment by levino** [comment]
> I'm doing it locally since upgrading the direct dependencies will inherently resolve most of the other ones, so it's a bit cleaner imo. Then I can proceed with doing the ones that impact the lock file only after.

Reasonable. However I would suggest to disable dependabot then. It only creates noise.

Where is the deployment of themes.jsonresume.org documented? If it is done via IaC, I might be able to fix it.

**Comment by SethFalco** [comment]
> However I would suggest to disable dependabot then. It only creates noise.

I'll hold off for two reasons:

* I only said my personal workflow, other maintainers may prefer to use it.
* It's also a good way to get notified… we'll just have to start actually doing it.

> Where is the deployment of themes.jsonresume.org documented? If it is done via IaC, I might be able to fix it.

I'm actually not sure. Maybe @thomasdavis could chime in?

**Comment by thomasdavis** [comment]
Hey @levino 

Welcome to the project. The CLI is so out of date in general I think it should just be rewritten at this point. 

I personally think the community alternatives are better;

https://github.com/rbardini/resumed
https://github.com/karanpargal/resume-cli

I am also slowly moving everything in to a monorepo -> https://github.com/jsonresume/jsonresume.org 

I think a new CLI tool should be fleshed out inside that repo.

But if you just want to fix up this one, I'm happy to give you collaborator access.

Re. dependabot, I actually don't like it so would be happy to see it go.

**Comment by levino** [comment]
If the community alternatives are better, why not embrace / continue them and properly deprecate this repo? At least [one](https://github.com/karanpargal/resume-cli#license) of them has MIT license. And I bet the other ones maintainers just forgot to add it. I would just archive this project here and link to one or both of these other projects. I for my part did not find these community alternatives before because it is not clearly said in the docs that it makes little sense to use the official one.

I had a look a the monorepo. Looks like fun but also a bit like a construction site and I cannot see the overall picture. If there are concrete tasks, I might help out a bit, if I can.

I also came here to fix #768 and just thought, before I can anything about that, it would make sense to fix some of the other obvious issues first. But I guess that issue can be closed as wontfix now and I will have a look if `resumed` or the other `resume-cli` support bun already.

Thank you for coming in and explaining the situation and saving me potentially a lot of time. I hope this conversation is also useful for future onlookers.
